receiver/cloudfoundryreceiver/README.md

```diff
@@ -0,0 +1,88 @@
+# Cloud Foundry Receiver
+
+The Cloud Foundry receiver connects to the RLP (Reverse Log Proxy) Gateway of the Cloud Foundry installation, typically
+available at the URL `https://log-stream.<cf-system-domain>`.
+
+## Authentication
+
+RLP Gateway authentication is performed by adding the Oauth2 token as the `Authorization` header. To obtain an OAuth2
+token to use for the RLP Gateway, a request is made to the UAA component which acts as the OAuth2 provider (URL
+specified by `uaa_url` configuration option, which typically is `https://uaa.<cf-system-domain>`). To authenticate with
+UAA, username and password/secret combination is used (`uaa_username` and `uaa_password` configuration options). This
+UAA user must have the `client_credentials` and `refresh_token` authorized grant types, and `logs.admin` authority.
+
+The following is an example sequence of commands to create the UAA user using the `uaac` command line utility:
+* `uaac target https://uaa.<cf-system-domain>`
```

Comment:
nit: `markdownlint` wants to have a blank line between a paragraph and a list

Reply:
Fixed.

Reply:
nit: the same comment applies to other places in the `README.md`